Week1/homework/app.js

```diff
@@ -1,11 +1,161 @@
+    },
+    the_brothers_karamazov: {
+      title: 'The Brothers Kazamazov',
+      author: ' Fyodor Dostoevsky',
+      genre: 'Philosophical novel',
+      language: 'Russian',
+      country: 'Russia',
+    },
+    in_search_of_lost_time: {
+      title: 'In Search of Lost Time',
+      author: ' Marcel Proust',
+      genre: 'Modernist',
+      language: 'French',
+      country: 'France',
+    },
+    great_expectations: {
+      title: 'Great Expectations',
+      author: ' Charles Dickens',
+      genre: 'Novel',
+      language: 'English',
+      country: 'United Kingdom',
+    },
+    war_and_peace: {
+      title: 'War and Peace',
+      author: ' Leo Tolstoy',
+      genre: 'Historical novel',
+      language: 'Russian',
+      country: 'Russia',
+    },
+    the_great_gatsby: {
+      title: 'The Great Gatsby',
+      author: 'F. Scott Fitzgerald',
+      genre: 'Novel',
+      language: 'English',
+      country: 'United States',
+    },
+    to_kill_a_mockingbird: {
+      title: 'To Kill a Mockingbird',
+      author: 'Harper Lee',
+      genre: ' Southern Gothic, Bildungsroman',
+      language: 'English',
+      country: 'United States',
+    },
+    the_adventures_of_huckleberry_finn: {
+      title: 'The Adventures of Huckleberry Finn',
+      author: 'Mark Twain',
+      genre: 'Picaresque novel',
+      language: 'English',
+      country: 'United States',
+    },
+  };
+
+  
+  const bookCovers = {    
+    harry_potter_and_the_chamber_of_secrets: './img/harry_potter_and_the_chamber_of_secrets.jpg',
+    the_art_of_war: ' ./img/the_art_of_war.jpg',
+    the_rise_and_fall_of_the_great_powers: ' ./img/the_rise_and_fall_of_the_great_powers.jpg',
+    the_brothers_karamazov: ' ./img/the_brothers_karamazov.jpg',
+    in_search_of_lost_time: './img/in_search_of_lost_time.jpg',
+    great_expectations: ' ./img/great_expectations.jpg',
+    war_and_peace: ' ./img/war_and_peace.jpg',
+    the_great_gatsby: ' ./img/the_great_gatsby.jpg',
+    to_kill_a_mockingbird: ' ./img/to_kill_a_mockingbird.jpg',
+    the_adventures_of_huckleberry_finn: ' ./img/the_adventures_of_huckleberry_finn.jpg',
+  };
+
+
+  const newDiv = document.createElement('div');
+  console.log(newDiv);
+  document.getElementsByTagName('body')[0].appendChild(newDiv);
+  const newH1 = document.createElement('h1');
+  console.log(newH1);
+
+  newDiv.appendChild(newH1);
+  newH1.innerText = "THE BEST 10 HIGH RANKED BOOKS EVER "
+
+
+  function generateMyFavoriteBookList() {
+
+    const newUl = document.createElement('ul');
+    console.log(newUl);
+    newDiv.appendChild(newUl);
+
+    const detailsOfBooks = Object.values(infoForEachBook);
+    console.log(detailsOfBooks);
+
+    for (let i = 0; i < detailsOfBooks.length; i++) {
+      const newLi = document.createElement('li');
+      newUl.appendChild(newLi);
+      const newImg = document.createElement('img');
+      newLi.appendChild(newImg);
+      newImg.src = Object.values(bookCovers)[i];
```

Comment:
On line 121 you are generating an array containing the values of the infoForEachBook:
```
    const detailsOfBooks = Object.values(infoForEachBook);
```
On line 129 you are doing something similar for the object of bookCovers. 
```
      newImg.src = Object.values(bookCovers)[i];
```
You are assuming the order of those arrays is the same. The homework says: "Remember that objects are not ordered, so you cannot guarantee that the first key is the first li element. (Hint: you could give each li item an id tag by modifying the function you made before.)"
Other than that you are now using those objects as if they where arrays and you are not at all using `bookTitles` anymore. 
The idea in this homework is to have an array with book titles, and two objects with information. You should loop over the array of book titles and look for the key in both objects to add the right information.